### ISSUE TITLE
Raise exception when using "--neuronwise_optimize" parameter

### ISSUE THREAD

**Issue by zhangzhaodi233** [ZhaoyangLyu/FROWN#1]
Hi Zhaoyang,

I'm really into your tool - FROWN, and I'm using it to get certifier bounds on my pretrained models with sigmoid. But when I used the parameter "--neuronwise_optimize". The code abrupt and raise exception in the following line.

https://github.com/ZhaoyangLyu/FROWN/blob/144065d009697cad95aa0c9c9a45a75bd6395877/get_bound_for_general_activation_function.py#L82-L84

Then, I read the paper "Fastened CROWN: Tightened Neural Network Robustness Certificates" again. In this paper, there is a Condition 1 - self-consistency. In the Chapter of "Fastened CROWN", it mentioned that "Condition 1 is too strong a condition for our proposed optimization framework to be practical. Actually we propose to improve CROWN by breaking Condition 1."

So, I don't know if the raised exception is related to the failure to meet Condition 1. 

Thanks and regards,
Zoey
